src/lib/read-models/time-to-production/time-to-production.ts

```diff
@@ -27,6 +29,7 @@ export class TimeToProduction {
         this.events = events;
     }
 
+    // todo: remove
```

Comment:
```suggestion
    /**
     * @deprecated TODO: remove
     */
```

Reply:
I'm hoping to remove those TODOs no later than next week so I don't invest time in them :)